components/keyboardScene.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component name="keyboardScene" extends="Scene" initialFocus="mainRowList">
+    <interface>
+        <field id="searchTerm" type="string" />
+    </interface>
+    <script type="text/brightscript" uri="keyboardScene.brs" />
+    <children>
+        <Group id="mainSearch">
+            <MiniKeyboard id="keyboard" translation="[150,150]" />
```

Comment:
To make this harder :smile: please make the keyboard component yourself. Make it look like the video.

Reply:
Alright

Reply:
- I tried numeros ways to fix the focus but it wont be fixed, I'll try again
- gonna fix the W part
- definitely possible 😉

Reply:
It is better now.

![2021-09-30-112948](https://user-images.githubusercontent.com/6665860/135417037-163ec6f0-9a49-4183-9e34-6eedbe85895a.jpg)

- `W` is not fixed.
- The result list are on top of the keyboard.

Reply:
Done